tests/algorithm/scripts/run_test.py

```diff
@@ -0,0 +1,34 @@
+import os
```

Comment:
```suggestion
#!/usr/bin/env python3

import os
```

Reply:
Will also need to run `chmod +x` on this file

Reply:
Added and also updated the readme to call out the `chmod +x` command